src/sky.cpp

```diff
@@ -202,15 +202,14 @@ void Sky::render()
 
 		driver->setMaterial(m_materials[1]);
 
-		video::SColor cloudyfogcolor = m_bgcolor;
-
-		// Draw far cloudy fog thing blended with skycolor
+		// Draw top band of our smooth skybox
```

Comment:
It's not the top band, the top band is from 0.3 up to 1.0.

Reply:
Looks incorrect with colours that have a reasonably large contrast:

Sides stretched to 1.0:
![screenshot_20181111_215526](https://user-images.githubusercontent.com/1439852/48318730-845ce580-e5fc-11e8-89d9-e760a5891863.png)

Sides kept at 0.3:
![screenshot_20181111_215633](https://user-images.githubusercontent.com/1439852/48318740-ae160c80-e5fc-11e8-8fb8-55298e5bb68f.png)

Sides lifted to 0.7:
![screenshot_20181111_215733](https://user-images.githubusercontent.com/1439852/48318755-ce45cb80-e5fc-11e8-95ce-d289802eaebd.png)

Reply:
It's trivial to change blending colours on the fly, but, having the blending done with pre-defined colours  (or supplied by set_sky or other) would be better.

Reply:
We could perhaps have skycolor gradient from horizon haze top to 0.3, then a darker blue from 0.3 to 1.0.

Reply:
That makes sense, however, doing so would interfere with rendering of fog, but would be easier. However, with the near cyanic colour of the current skies, darkening it would make it look horrendous. I'd say bump the height of the side to 0.65, then change actual sky colours later on to look cleaner and less comic-like.

Reply:
Ok, this discussion isn't much of an issue anyway, marking as resolved :)